### DISCUSSION TITLE
How to get all payment methods on Shop-api?

### DISCUSSION THREAD

**Discussion by vankhoadesign** [vendurehq/vendure#721] [Q&A]
I want to show all payment methods on checkout page. So i can't found api for that.

**Comment by michaelbromley** [comment]
There is a new query `eligiblePaymentMethods` which works just like the existing `eligibleShippingMethods`, coming in the next release: https://github.com/vendure-ecommerce/vendure/commit/e528c099feccb9581800b91a5d6c221af03c3650
